src/client-websocket.ts

```diff
@@ -0,0 +1,11 @@
+declare let window: any;
+let _global = typeof global !== 'undefined' ? global : (typeof window !== 'undefined' ? window: {});
+
+import * as NodeSocket from 'ws';
```

Comment:
So the old websocket package has a `browser` field in its package.json which points to a tiny file which just exports the native WebSocket object.  This means that when, eg, webpack (probably Meteor too, I only tested with GitHunt-react) bundles up subscriptions-transport-ws, it won't actually include all of the Node-oriented code from websocket in the client bundle.

That doesn't seem to work here: `ws` doesn't have the browser field so you're always going to bundle the entire `ws` library into the client bundle even though none of it is used.  This seems bad.  I think this would happen even if you converted the import into a require and put it under a conditional?

I think you can probably fix this by using the "browser" field yourself in this package's package.json somehow?  I know we want to be able to use Client on the server too (for tests if nothing else) but there's probably some way to do it that keeps code splitting working.